docs/java-troubleshooting.md

```diff
@@ -0,0 +1,31 @@
+---
+id: java-troubleshooting
+title: Scope Java Agent troubleshooting
+sidebar_label: Troubleshooting
+---
+
+## Problem
+
+You do not see data in Scope after [installing the Scope Java agent](java-installation.md).
+
+## Solution
+
+### Have you configured a valid `SCOPE_DSN`?
+
+The Scope Java agent needs a valid `SCOPE_DSN` configured as environment variable.
+
+Keep in mind that `SCOPE_DSN` is different per **namespace** so if you configure a `SCOPE_DSN` from other namespace, although it was from the correct environment, you will not see results in Scope for that repository.
```

Comment:
`although it was from the correct environment`

What do you mean?

Reply:
I tried to express the fact about "if you are in the correct Scope instance", like SaaS or your on prem environment, it is not sufficient.

Reply:
I think the likelihood of someone using different Scope instances at the same time is very low, so we can just remove that and should be fine. Most people will only use one Scope instance, either the SaaS one or their on-prem one.